config/logstash.yml

```diff
@@ -216,6 +216,23 @@
 # Default is false
 # pipeline.separate_logs: false
 #
+# Enables internal collector to shipping directly to Elasticsearch
+#monitoring.enabled: false
+#monitoring.elasticsearch.username: logstash_system
+#monitoring.elasticsearch.password: password
+#monitoring.elasticsearch.hosts: ["https://es1:9200", "https://es2:9200"]
+# an alternative to hosts + username/password settings is to use cloud_id/cloud_auth
+#monitoring.elasticsearch.cloud_id: monitoring_cluster_id:xxxxxxxxxx
+#monitoring.elasticsearch.cloud_auth: logstash_system:password
+#monitoring.elasticsearch.ssl.certificate_authority: [ "/path/to/ca.crt" ]
+#monitoring.elasticsearch.ssl.truststore.path: path/to/file
+#monitoring.elasticsearch.ssl.truststore.password: password
+#monitoring.elasticsearch.ssl.keystore.path: /path/to/file
+#monitoring.elasticsearch.ssl.keystore.password: password
+#monitoring.elasticsearch.ssl.verification_mode: certificate
+#monitoring.elasticsearch.sniffing: false
+#monitoring.collection.interval: 10s
+#monitoring.collection.pipeline.details.enabled: true
 # ------------ X-Pack Settings (not applicable for OSS build)--------------
```

Comment:
I think we might want to consider removing the `xpack` settings here, and just have the `monitoring` settings. We should also make a note to surface this in release notes and blogs for the `7.7.0` release